test_responses.py

```diff
@@ -922,3 +922,18 @@ def test_custom_target(monkeypatch):
     requests_mock.start()
     assert len(patch_mock.call_args_list) == 1
     assert patch_mock.call_args[1]["target"] == "something.else"
+
+
+def _quote(s):
+    return responses.quote(responses._ensure_str(s))
+
+
+def test_cookies_from_headers():
+    text = "こんにちは/世界"
+    quoted_text = _quote(text)
+    expected = {"x": "a", "y": quoted_text}
+    headers = {"set-cookie": "; ".join(k + "=" + v for k, v in expected.items())}
+    cookiejar = responses._cookies_from_headers(headers)
+    for k, v in cookiejar.items():
+        assert isinstance(v, str)
```

Comment:
Do you want `str` instances in python2?

Reply:
That's the kind of design decision that I'm not exactly sure of, especially given that `__future__.unicode_literals` is  imported in the module.

My thinking is that `str` is more compatible with how cookies works in `requests` in Python 2.7, which simply store them as `str`, and that makes sense given that they contain URL-quoted chars, not non-ASCII chars, and for that the type doesn't need to be `unicode`.